src/Symfony/EventListener/WriteListener.php

```diff
@@ -64,15 +67,21 @@ public function onKernelView(ViewEvent $event): void
             return;
         }
 
-        if (!($operation?->canWrite() ?? true) || !$attributes['persist']) {
+        if (!$attributes['persist'] || !($operation?->canWrite() ?? true)) {
             return;
         }
 
         if (!$operation?->getProcessor()) {
             return;
         }
 
-        $context = ['operation' => $operation, 'resource_class' => $attributes['resource_class'], 'previous_data' => $attributes['previous_data'] ?? null];
+        $context = [
+            'operation' => $operation,
+            'resource_class' => $attributes['resource_class'],
+            'read_data' => $attributes['read_data'] ?? null,
```

Comment:
is `read_data` really needed?

Reply:
Yes because we need a reference to the original fetched data (not a clone). And it is lost in DeserializeListener is the serializer creates a new object.

That being said I didn't try with the clone, but I double it works (I'll check).

Reply:
I've found a hack to get rid of it.